### ISSUE TITLE
Code structure

### ISSUE THREAD

**Issue by dilab** [adamkopec/ddd_cart#1]
Did you follow any particular reference to create the folder structure of your project?

**Comment by adamkopec** [comment]
Not really, it's based on my personal preference. I think that the most important thing is separation of publicly accessible files (public/) from the application code (application/). If you have some more specific questions, e.g. you found something counterintuitive - I'd try to do my best to explain myself :)
